mars-sim-core/src/main/java/com/mars_sim/core/person/NationSpecConfig.java

```diff
@@ -20,28 +21,25 @@
  */
 public class NationSpecConfig extends UserConfigurableConfig<NationSpec> {
 
-	private final String COUNTRY_XSD = "country.xsd";
+	private static final String COUNTRY_XSD = "country.xsd";
 	
 	private static final String COUNTRY = "country";
 
-	private final String ECONOMICS_DATA = "economics-data";
-//	private final String GDP = "GDP";
```

Comment:
These figures (GPD, PPP, etc.) will be put into use in simulating the economic potential of a nation. 

All country xml (such as country_austria.xml) files have these 4 basic figures already.

Reply:
They are still loaded. There is a ug because the crews are using USA but that no longer exists. I've fixed it